site/content/en/docs/FAQ/_index.md

```diff
@@ -79,6 +79,11 @@ through [SDK.WatchGameServer()]({{< ref "/docs/Guides/Client SDKs/_index.md#watc
 Combining these two features allows you to pass information such as map data, gameplay metadata and more to a game
 server binary at Allocation time, through Agones functionality.
 
+Do note, that if you wish to have either the labels or annotations that are set via `GameServerAllocation` to be 
+editable by the game server via the Agones SDK, the label key will need to be prefixed with `agones.dev/sdk-`. See
```

Comment:
this doesnt read right in my head at least...

```suggestion
Do note, that if you wish to have either the labels or annotations set by `GameServerAllocation` to ensure these are editable by the game server from the Agones SDK, the label key will need to be prefixed with `agones.dev/sdk-`. See
```

Reply:
Additional comment ... do we refer to gs as `gameservers` `game server` `Game Server`? I tend to refer to it as DGS dedicated game server :joy:

Reply:
I usually use `GameServer` when referring to the CRD, and game server when talking about the binary.

Reply:
Tweaked the wording a bit - I think I solved the awkward phrasing. Thanks!